File: PRAK205/PRAK205-2210817310015-RIDHANISETIADI.py
import math
A=int(input("Masukkan nilai pertama :"))
B=int(input("Masukkan nilai kedua :"))
C=math.sqrt((B*B)-(A*A))
print("\nAlas= ",end= "")
print("%d" %C, "cm")
print("Tinggi =",A,"cm")
K=A+B+C
print("Keliling =",round(K),"cm")
L=(1*A*C) / (2)
print("Luas =",round(L),"cm^2")

File: PRAK205/PRAK205-2210817310015-RIDHANISETIADI.c
#include<stdio.h>
#include <math.h>
int main()
{
    int A,B,C,K,L;
    printf("Masukkan nilai  A = ");scanf("%d",&A);
    printf("Masukkan nilai  B = ");scanf("%d",&B);
    C=sqrt((B*B)-(A*A));
    printf("\nAlas= %d cm",C);
    printf("\nTinggi = %d cm",A);
    K=(A+B+C);
    printf("\nKeliling = %d cm",K);
    L=(0.5*A*C);
    printf("\nLuas = %d cm^2",L);
}
